com.unity.sg2/Documentation~/Imposter-UV-Node.md

```diff
@@ -0,0 +1,71 @@
+# Imposter UV Node
+
+## Description
+
+The Imposter UV Node calculates the billboard position and the UV coordinates needed by the Imposter Sample node.
+## Ports
+
+| Name        | Direction           | Type  | Description |
+|:------------ |:-------------|:-----|:---|
+| In Position | Input      |    Vector3 | The postion in Object space |
+| In UV | Input      |    Vector4 | The UV coordinates of the mesh |
+| Frames | Input      |    Float | The number of the imposter frames in each axis|
+| Offset | Input      |    Float | The offset value from the pivot |
+| Size | Input      |    Float | The size of the imposter |
+| Height map | Input      |    Texture2D | The height map texture to sample |
+| Sampler | Input      |    Sampler State | The texture sampler to use for sampling the texture |
+| Parallax | Input      |    Float | Parallax strength|
```

Comment:
We need to make it clear that we're not doing any parallax if this is set to 0 and the other parallax-related controls won't do anything if this is zero.

Reply:
good point!